src/internal/clusterstate/v2.11.0/clusterstate.go

```diff
@@ -9,5 +9,6 @@ func Migrate(state migrations.State) migrations.State {
 		Apply("Normalize commit totals", normalizeCommitTotals, migrations.Squash).
```

Comment:
Totally unrelated to this PR.  The first migration of the release should not be squashed.

Reply:
Good catch. I can update that as part of this PR.